website/blog/2023-11-11-AgentEval/index.mdx

```diff
@@ -0,0 +1,144 @@
+
+---
+title:  AgentEval: Assessing Task Utility for LLM-powered Applications
+authors: julianakiseleva, narabzad

```

Comment:
```suggestion
title: AgentEval for Assessing Task Utility for LLM-Powered Applications
authors: 
  - julianakiseleva
  - narabzad
```
docusaurus complains with ":" in the title
"P" should be capitalized
add correct format for multiple authors

Reply:
@gagb  Thanks for digging into this! Incorporating this fix to the next version.